src/components/NavBar/NavBar.style.ts

```diff
@@ -147,16 +152,18 @@ export const GridItemLogo = styled.div<{ hasScrolled: boolean }>`
 
 export const GridItemSearch = styled.div<{ hasScrolled: boolean }>`
   ${row};
+  // On mobile, search expands to also use the logo column after scrolling.
   grid-column: ${({ hasScrolled }) => (hasScrolled ? '2/4' : '3/4')};
 
   @media (min-width: ${mobileBreakpoint}) {
-    grid-column: 1/6;
+    grid-column: 3/4;
```

Comment:
This is the main fix.  I made the search bar only use the 3rd column instead of covering columns 2-5.

Reply:
ooo, i originally set it to span all columns on desktop so that it would be centered within the whole nav bar, not just within its allotted space-- but 3/4 seems to accomplish that 🙌🏼 🤷🏻‍♀️

Reply:
oh actually, jk, its thrown off slightly // no longer centered within the navbar as whole. but its not really noticeable, and getting the navbar fully functional is more important

Reply:
Aha! I see, yeah, that makes sense.  Thanks for explaining. I'll go forward with this fix but I guess if we wanted to keep the prior behavior we probably could, we'd just need to start playing with z-order to make sure the buttons end up on top of the search bar.